### PR TITLE
Tasks 877-881: kde4, schema_guided_dstc8 Datasets

### PR DESCRIPTION
**Pull request by kashyap467**

### CODE REVIEW

File: tasks/task880_schema_guided_dstc8_classification.json [hunk 1]
```diff
@@ -0,0 +1,8106 @@
+{
+    "Contributors": ["Subba Raja Kashyap Saligrama"],
+    "Source": ["schema_guided_dstc8 (https://huggingface.co/datasets/schema_guided_dstc8)"],
+    "Categories": ["Classification"],
+    "Definition": "You are given a sentence from a conversation between a human and a virtual assistant. Your task is to classify the sentence into one of following five action categories - INFORM, INFORM_INTENT, OFFER, REQUEST, REQUEST_ALTS",
```
Comment (danyaljj) [inline tasks/task880_schema_guided_dstc8_classification.json]: I am not sure if the definition is clear. 

How are `INFORM` and `INFORM_INTENT`? 
Similarly, `REQUEST` and  `REQUEST_ALTS`? (what does `ALTS` stand for?)

**Comment by danyaljj** [comment]
Overall the tasks look good. For one of the tasks (as suggested above), you need to elaborate on the task definition.

**Comment by aarunku5** [comment]
> Overall the tasks look good. For one of the tasks (as suggested above), you need to elaborate on the task definition.

@kashyap467 can you update the tasks accordingly? Thanks!

**Comment by danyaljj** [comment]
Please make sure to set the `"Categories": [""]` and `"Domains": [""]` fields according to [the taxonomy presented here](https://github.com/allenai/natural-instructions-expansion/blob/master/doc/task-hierarchy.md).

**Pull request merged**